### PR TITLE
fixed create department and find department endpoints

### PR DESCRIPTION
**Pull request by MariyaSin**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/754

### CODE REVIEW

**Review by TIove — APPROVED** [review]

File: src/CompanyService.Models.Db/DbDepartment.cs [hunk 1]
```diff
@@ -37,7 +37,8 @@ public void Configure(EntityTypeBuilder<DbDepartment> builder)
 
             builder
                 .HasMany(d => d.Users)
-                .WithOne(u => u.Department);
+                .WithOne(u => u.Department)
+                .HasForeignKey(k => k.DepartmentId);
```
Comment (BelehovEgor) [inline src/CompanyService.Models.Db/DbDepartment.cs:41]: this key have to be in DepartmentUser not here. DbDepartment is main entity, DepartmentUser dependent

Comment (djino97) [inline src/CompanyService.Models.Db/DbDepartment.cs:41]: Not need HasForeignKey

File: src/CompanyService.Models.Dto/Configuration/RabbitMqConfig.cs [hunk 2]
```diff
@@ -19,6 +18,9 @@ public class RabbitMqConfig : BaseRabbitMqConfig
         public string ChangeUserPositionEndpoint { get; set; }
 
         [AutoInjectRequest(typeof(IGetUsersDataRequest))]
+        public string GetUsersDataEndpoint { get; set; }
+
+        [AutoInjectRequest(typeof(IGetPositionRequest))]
```
Comment (djino97) [inline src/CompanyService.Models.Dto/Configuration/RabbitMqConfig.cs:23]: Not need AutoInjectRequest

File: src/CompanyService/Controllers/DepartmentController.cs [hunk 3]
```diff
@@ -2,21 +2,39 @@
...
         [HttpPost("create")]
-        public Guid Create(
+        public OperationResultResponse<Guid> Create(
             [FromServices] ICreateDepartmentCommand command,
             [FromBody] CreateDepartmentRequest department)
         {
-            return command.Execute(department);
+            var result = command.Execute(department);
+
+            if (result.Status != Kernel.Enums.OperationResultStatusType.Failed)
```
Comment (djino97) [inline src/CompanyService/Controllers/DepartmentController.cs:32]: Why not use namespace for Kernel.Enums.OperationResultStatusType?

**Review by djino97 — APPROVED** [review]

File: src/CompanyService.Models.Dto/Configuration/RabbitMqConfig.cs (hunk 2 above)
Comment (MariyaSin) [reply src/CompanyService.Models.Dto/Configuration/RabbitMqConfig.cs:23]: done

File: src/CompanyService.Models.Db/DbDepartment.cs (hunk 1 above)
Comment (MariyaSin) [reply src/CompanyService.Models.Db/DbDepartment.cs:41]: done

Comment (MariyaSin) [reply src/CompanyService.Models.Db/DbDepartment.cs:41]: done

File: src/CompanyService/Controllers/DepartmentController.cs (hunk 3 above)
Comment (MariyaSin) [reply src/CompanyService/Controllers/DepartmentController.cs]: done

**Review by BelehovEgor — APPROVED** [review]

**Pull request merged**